packages/semantic-ui/src/SelectWidget/SelectWidget.js

```diff
@@ -6,9 +6,7 @@ import { utils } from '@rjsf/core';
 import PropTypes from "prop-types";
 import { getSemanticProps } from "../util";
 
-const { asNumber, guessType } = utils;
-
-const nums = new Set(["number", "integer"]);
+const { processNewValue } = utils;
```

Comment:
It happens the `processValue()` function was not even implemented in the `semantic-ui` widget. But maybe for a reason?

Reply:
What do you mean? It seems like `processValue` was implemented in the semantic-ui widget -- see the diff for this file.

Reply:
Sorry, I misread: the function was not originally part of `fluent-ui`.

Reply:
👍